hack/tools.go

```diff
@@ -0,0 +1,26 @@
+//go:build tools
+
+// Copyright 2023 The Kubeflow Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tools
```

Comment:
what's this for?

Reply:
I referred to the following:

https://github.com/golang/go/wiki/Modules#how-can-i-track-tool-dependencies-for-a-module